third_party/lua/luadebug.lua

```diff
@@ -34,8 +34,9 @@ local COLOR_RED = ""
 local COLOR_BLUE = ""
 local COLOR_YELLOW = ""
 local COLOR_RESET = ""
-local GREEN_CARET = " => "
-local RED_BREAK = "  ● "
+local GREEN_CARET_SYM = "=>"
```

Comment:
Why is it a `GREEN` caret sym? Wouldn't it be nicer to name it just `CARET` as it is always a caret and not always green (when color is not supported or is turned off explicitly)?

Reply:
`_SYM` because it's caret alone, dislike in other cases when it was surrounded by spaces.

Reply:
That's perfectly clear. The question is why your caret is not green here. There is a case, when NO_COLOR is set by env and the `GREEN_CARET_SYM` contains plain caret symbol for the whole execution. I don't think that green is the primary characteristic of caret as it can be green or plain, but not red.

Reply:
> `GREEN_CARET` (that can actually be plain) is colored yellow.

Nope, it's not. Yellow there is only `DEBUGGER` (that's the whole purpose of `color_*()` wrapping functions - they begin color via corresponding escape sequence, concatenate given text, and then reset color back)